frontend/packages/kubevirt-plugin/src/components/vms/vm-environment/vm-environment-page.tsx

```diff
@@ -377,6 +376,7 @@ const VMEnvironment = withHandlePromise<VMEnvironmentProps>(
             firstTitle="configmap / secret / service account"
             secondTitle="Serial Number"
             addButtonDisabled={addButtonDisabled || inProgress}
+            addButtonLabel="Add Configmap, Secret or Service Account"
```

Comment:
shouldn't this rather be like this ?
```suggestion
            addButtonLabel="Add Config Map, Secret or Service Account"
```